ci/docker/serve.build.Dockerfile

```diff
@@ -26,5 +26,8 @@ RUN pip install -U --ignore-installed \
 
 RUN git clone https://github.com/wg/wrk.git /tmp/wrk && pushd /tmp/wrk && make -j && sudo cp wrk /usr/local/bin && popd
 
-# Install custom Pydantic version if requested.
-RUN if [[ -z $PYDANTIC_VERSION ]] ; then echo Not installing custom Pydantic version ; else pip install -U pydantic==$PYDANTIC_VERSION ; fi
+# Install Pydantic from source if requested.
+RUN if [[ -z $PYDANTIC_BUILD_COMMIT ]] ; \
+    then echo "Not installing Pydantic from source." ; \
+    else git clone https://github.com/pydantic/pydantic && pushd pydantic && git checkout $PYDANTIC_BUILD_COMMIT && pip install -e . && popd ; \
```

Comment:
Non-blocker: We can probably just do ` pip install -U git+https://github.com/pydantic/pydantic@$PYDANTIC_BUILD_COMMIT` to install on a particular commit 🙃

Reply:
Oh, you're totally right...